src/styles/dropdownMenu.css

```diff
@@ -8,7 +8,7 @@
     z-index: var(--z-5);
     position: absolute;
     top: 4rem;
-    right: 14.6rem;
+    right: 12.6rem;
     background-color: #ecfccb;
     border: 1px solid var(--lime-700);
     padding: 1.2rem 1.35rem;
```

Comment:
Its a good practice to use sizes in the multiples of 4 or 8px for design consistency.